homeassistant/components/shelly/bluetooth/scanner.py

```diff
@@ -10,7 +9,7 @@
 from homeassistant.components.bluetooth import BaseHaRemoteScanner
 from homeassistant.core import callback
 
-_LOGGER = logging.getLogger(__name__)
```

Comment:
I generally prefer having a logger per module since its easier to grep for the specific module when debugging